tests/unit/test_charm.py

```diff
@@ -49,7 +53,7 @@ def test_config_changed(self, _define_service, _define_ingress, _report_service_
         # Confirm status is as expected.
         self.assertEqual(
             self.harness.charm.unit.status,
-            ActiveStatus("Ingress with service IP(s): 10.0.1.12"),
+            ActiveStatus("Ingress IP(s): 10.0.1.12, Service IP(s): 10.0.1.13"),
```

Comment:
Tests that check for specific strings on informational messages like this can be flaky and require duplicating the message in the source and test code. A potential alternative is to look for key words, word sequences or values in the string. For example, this assert could look for that the message contains the required IP addresses

Reply:
I don't think they'd be any more flaky than checking for specific parts of the message. It does mean you might need to update the tests if you update the message, but I don't think that's a bad thing. Personally I prefer seeing more verbose messages in tests because it's easier to understand what you're testing for, and confirms the wording is correct, which can sometimes be important. If you're just looking for a specific IP address here how do you know if it's the ingress address or the service address?